lib/logflare/source/recent_logs_server.ex

```diff
@@ -240,11 +241,18 @@ defmodule Logflare.Source.RecentLogsServer do
     end
   end
 
+  def handle_info({:EXIT, _from, _reason}, state) do
```

Comment:
```suggestion
  def handle_info({:EXIT, _from, _reason}, _state) do
```

Reply:
Huh? Don't I need state in my :noreply tuple?

Reply:
Maybe you were meaning the terminate...